### PR TITLE
[Merged by Bors] - chore: backports from nightly-testing-2024-08-12

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/RingTheory/Coprime/Lemmas.lean [hunk 1]
```diff
@@ -151,8 +151,9 @@ theorem exists_sum_eq_one_iff_pairwise_coprime [DecidableEq I] (h : t.Nonempty)
       use fun i ↦ if i = a then u else v * μ i
       have hμ' : (∑ i ∈ t, v * ((μ i * ∏ j ∈ t \ {i}, s j) * s a)) = v * s a := by
         rw [← mul_sum, ← sum_mul, hμ, one_mul]
-      rw [sum_cons, cons_eq_insert, sdiff_singleton_eq_erase, erase_insert hat, if_pos rfl,
-        ← huv, ← hμ', sum_congr rfl]
+      rw [sum_cons, cons_eq_insert, sdiff_singleton_eq_erase, erase_insert hat]
+      simp only [↓reduceIte, ite_mul]
+      rw [← huv, ← hμ', sum_congr rfl]
```
Comment (semorrison) [inline Mathlib/RingTheory/Coprime/Lemmas.lean:156]: This fails on `nightly-testing`, I haven't actually diagnosed why.

Comment (semorrison) [reply Mathlib/RingTheory/Coprime/Lemmas.lean:156]: Happy to take this diff out (or for anyone else to do so).

Comment (jcommelin) [reply Mathlib/RingTheory/Coprime/Lemmas.lean:156]: Whichever you think is most convenient...

**Review by jcommelin — COMMENTED** [review]
bors d+

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**